src/libostree/ostree-repo-commit.c

```diff
@@ -412,23 +412,9 @@ _ostree_repo_open_content_bare (OstreeRepo          *self,
                                 const char          *checksum,
```

Comment:
Can we update the preceding comment block too?

Reply:
Yep done ⬇️